### ISSUE TITLE
Final Report Review

### ISSUE THREAD

**Issue by liufan2312** [yuanchengl/ORIE4741-Seimic-Analysis-Project#6]
The final report is well-written and the dataset you were working on is very interesting. However, I think there is still room for improvement or further investigation.
<li>First of all, it will be really nice if you can put your code in this repo as well :)</li>
<li>You applied linear regression model to predict EDP for every entry of seismic input and you got some really promising results. However, in the report, you did not mention what error metrics you are using (is it mean absolute error? mean square error?). And it's hard for readers like me who do not have much background knowledge to understand if 0.00403 (test error for X_2 input) is good or bad. (I assume it's really good?) </li>
<li>Also, it will be nice to include training error side by side with testing error so that readers can understand how well the model is fitting for both training set and test set and whether it is overfitting or underfitting. Since you mention that it is inherently a non linear system, it will be interesting to investigate how well linear models fit the training data.</li>
<li>For input space X_1 and X_3, since there are more columns than rows, some dimensionality reduction techniques can be applied here like PCA not only to make the model run faster but also to avoid overfitting.</li>
<li>When building the input space, another question that I had is: would it be better fit you use X_1, X_2 and X_3 all together as input instead of using only one of them to fit the model? I would definitely tried to use all the features together and see if the model can achieve a better result.</li>
<br>But overall, I think you guys did a great job on this project and clearly you applied a lot of the knowledge covered in the lectures. Good luck with the rest of your exams!
